api/v1alpha1/volumesnapshotbackup_types.go

```diff
@@ -28,6 +29,22 @@ type VolumeSnapshotBackupSpec struct {
 	ProtectedNamespace string `json:"protectedNamespace,omitempty"`
 	// Restic Secret reference for given BSL
 	ResticSecretRef corev1.LocalObjectReference `json:"resticSecretRef,omitempty"`
+	// StorageClassName can be used to override the StorageClass of the source PVC
+	//+optional
+	StorageClassName string `json:"storageClassName,omitempty"`
```

Comment:
Would it make sense to bunch all these attributes together in some struct ?

Reply:
Yes because this is currently a little confusing. We should probably just make this `MoverConfiguration` or something so that its clear you are defining the storageclass/access modes to be used by the mover itself.

Reply:
In fact... why not just pull this spec type directly from volsync? Let the user override ANY of the volsync spec fields?

Reply:
I think the only remaining 2 are `retain` for the restic repo, and `volumeSnapshotClassName`. However `volumeSnapshotClassName` is only used for the `snapshot` copy method, which we do not use. But I can add in `retain`

Reply:
@eemcmullan in case I confused you, my thought is why not just expose a new field `ResticMoverOptions` and give the user full capability to configure any of the restic mover config values that go on replicationsource.

There MIGHT be downsides to this, just something I want us to consider

Reply:
downside would be volsync deciding to change its struct in the future and along the way forcing our CRD to change.